pkgs/applications/networking/instant-messengers/dorion/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  dpkg,
+  fetchurl,
+  autoPatchelfHook,
+  wrapGAppsHook,
+  webkitgtk,
+  glib-networking,
+  libappindicator,
+  libayatana-appindicator,
+  gst_all_1,
+}:
+stdenv.mkDerivation rec {
+  name = "dorion";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/SpikeHD/Dorion/releases/download/v${version}/Dorion_${version}_amd64.deb";
+    sha256 = "sha256-IYqJ5mz+XGHf4GVSW2Mq/z8xWLs4Y4KRWZ6fAtIg2tk=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.